src/components/Article.jsx

```diff
@@ -2,24 +2,44 @@ import '../App.css';
 import { useEffect, useState } from 'react';
 import { useParams } from 'react-router';
 import { getArticle } from '../utils/api';
+import { increaseVote } from '../utils/api';
+import { decreaseVote } from '../utils/api';
 
 const Article = () => {
     const [article, setArticle] = useState({});
+    const [votes, setVotes] = useState(0); 
     const [isLoading, setIsLoading] = useState(true);
     const {article_id} = useParams();
 
     useEffect(() => {
         getArticle(article_id)
         .then(({article}) => {
             setArticle(article)
+            setVotes(article.votes)
             setIsLoading(false);
         })
         .catch(err => {
             console.log(err);
         })
     }, [article_id]);
 
-    if (isLoading) return <p className='loading'>Loading...</p>   
+    if (isLoading) return <p className='loading'>Loading...</p>  
+
+    const voteOnArticle = (article_id) => {
+        setVotes((currVotes) => currVotes + 1)
+        increaseVote(article_id)
+        .then(( {data} ) => {
+            console.log( {data} )
+        })
```

Comment:
Technically don't need this .then() block because in essence it's doing nothing other than console logging, which isn't something we would want a user to see.

Reply:
I'd recommend removing this .then block or replacing it with a .catch() incase there was any error with the function not updating the database as you expected, you may want the .catch to revert the optimistic rendering back to -1?